include/odp/api/spec/init.h

```diff
@@ -355,6 +355,17 @@ int odp_term_global(odp_instance_t instance);
  */
 void odp_log_thread_fn_set(odp_log_func_t func);
 
+/**
+ * Get instance handle
+ *
+ * Returns the calling thread's ODP instance handle.
+ *
+ * @return Instance handle
+ *
+ * @see odp_init_global(), odp_init_local()
+ */
+odp_instance_t odp_instance(void);
```

Comment:
Agreed to change instance from return value to an output param. So, that global_init() and this call can output the same content into instance object.

Reply:
Fixed in V3.